docs/Config.md

```diff
@@ -30,6 +32,94 @@
     days: 14 # how often an update is checked for
   reporting: 'undetermined' # one of: 'on' | 'off' | 'undetermined'
   confirmOnQuit: false
+  keybinding:
+    universal:
+      quit: 'q'
+      quit-alt1: '<c-c>' # alternative/alias of quit
+      return: '<esc>' # return to previous menu, will quit if there's nowhere to return
+      quitWithoutChangingDirectory: 'Q'
+      togglePanel: '<tab>' # goto the next panel
+      prevItem: '<up>' # go one line up
+      nextItem: '<down>' # go one line down
+      prevItem-alt: 'k' # go one line up
+      nextItem-alt: 'j' # go one line down
+      prevBlock: '<left>' # goto the previous block / panel
+      nextBlock: '<right>' # goto the next block / panel
+      prevBlock-alt: 'h' # goto the previous block / panel
+      nextBlock-alt: 'l' # goto the next block / panel
+      optionMenu: 'x' # show help menu
+      optionMenu-alt1: '?' # show help menu
+      select: '<space>'
+      remove: 'd'
+      new: 'n'
+      edit: 'e'
+      openFile: 'o'
+      scrollUpMain: '<pgup>' # main panel scrool up
+      scrollDownMain: '<pgdown>' # main panel scrool down
+      scrollUpMain-alt1: 'K' # main panel scrool up
+      scrollDownMain-alt1: 'J' # main panel scrool down
+      scrollUpMain-alt2: '<c-u>' # main panel scrool up
+      scrollDownMain-alt2: '<c-d>' # main panel scrool down
+      createRebaseOptionsMenu: 'm'
+      pushFiles: 'P'
+      pullFiles: 'p'
+      refresh: 'R'
+      createPatchOptionsMenu: '<c-p>'
+    status:
+      checkForUpdate: 'u'
+      recentRepos: 's'
+    files:
+      commitChanges: 'c'
+      commitChangesWithoutHook: 'w' # commit changes without pre-commit hook
+      amendLastCommit: 'A'
+      commitChangesWithEditor: 'C'
+      ignoreFile: 'i'
+      refreshFiles: 'r'
+      stashAllChanges: 's'
+      viewStashOptions: 'S'
+      toggleStagedAll: 'a' # stage/unstage all
+      viewResetOptions: 'D'
+      fetch: 'f'
+      executeCustomCommand: 'X'
+    branches:
+      createPullRequest: 'o'
+      checkoutBranchesByName: 'c'
+      forceCheckoutBranch: 'F'
+      rebaseBranch: 'r'
+      mergeIntoCurrentBranch: 'M'
+      FastForward: 'f' # fast-forward this branch from its upstream
+      pushTag: 'P'
+      nextBranchTab: ']'
+      prevBranchTab: '['
+      setUpstream: 'u' # set as upstream of checked-out branch
+    commits:
+      squashDown: 's'
+      renameCommit: 'r'
+      renameCommitWithEditor: 'R'
+      resetToThisCommit: 'g'
+      fixupCommit: 'f'
+      createFixupCommit: 'F' # create fixup commit for this commit
+      squashAboveCommits: 'S'
+      moveDownCommit: '<c-j>' # move commit down one
+      moveUpCommit: '<c-k>' # move commit up one
+      amendToCommit: 'A'
+      pickCommit: 'p' # pick commit (when mid-rebase)
+      revertCommit: 't'
+      cherryPickCopy: 'c'
+      cherryPickCopyRange: 'C'
+      pasteCommits: 'v'
+      viewCommitFiles: '<enter>'
```

Comment:
I think this could be made universal. Basically whenever we want to go one level deeper in a panel we'll want to press enter or whatever this key is set to.

Reply:
I put `universal.goInto` and added a description in the documentation. It’s mapped to `enter` by default.